### PR TITLE
docs: add last replica timeout behavior to important notes

### PR DESCRIPTION
**Pull request by ejweber**
#### Which issue(s) this PR fixes:

longhorn/longhorn#8711

#### What this PR does / why we need it:

Follow up to https://github.com/longhorn/website/pull/978#issuecomment-2321588862 to address:

- https://github.com/longhorn/website/pull/978#issuecomment-2319481190
- https://github.com/longhorn/website/pull/978#pullrequestreview-2271431204

### CODE REVIEW

**Comment by derekbit** [comment]
cc @jillian-maroket

**Review by innobead — APPROVED** [review]
Technically LGTM. But wiring for reviews by doc engineer.

P.S. @derekbit we probably should start thinking of how different behaviors are managed in v1 mad v2 in the future.

File: content/docs/1.7.1/important-notes/_index.md [hunk 1]
```diff
@@ -159,6 +160,12 @@ RWX Volumes fast failover is introduced in Longhorn v1.7.0 to improve resilience
 
 Starting with v1.7.0, Longhorn supports configuration of timeouts for replica rebuilding and snapshot cloning. Before v1.7.0, the replica rebuilding timeout was capped at 24 hours, which could cause failures for large volumes in slow bandwidth environments. The default timeout is still 24 hours but you can adjust it to accommodate different environments. For more information, see [Long gRPC Timeout](../references/settings/#long-grpc-timeout).
 
+### Change in Engine Replica Timeout Behavior
+
+In versions before v1.7.1, the [Engine Replica Timeout](../references/settings#engine-replica-timeout) setting applied
+to all of a v1 volume's replicas equivalently. Now, a v1 engine will not mark the last active replica as failed until
+twice the configured number of seconds have passed.
```
Comment (jillian-maroket) [inline content/docs/1.7.1/important-notes/_index.md:167]:
```suggestion
In versions earlier than v1.7.1, the [Engine Replica Timeout](../references/settings#engine-replica-timeout) setting was equally applied to all V1 volume replicas. In v1.7.1, a V1 engine marks the last active replica as failed only after twice the configured number of seconds (timeout value x 2) have passed.
```

File: content/docs/1.7.1/references/settings.md [hunk 2]
```diff
@@ -424,9 +424,9 @@ The default minimum number of backing image copies Longhorn maintains.
 The time in seconds a v1 engine will wait for a response from a replica before marking it as failed. Values between 8
 and 30 are allowed. The engine replica timeout is only in effect while there are I/O requests outstanding.
 
-This timeout only applies as-configured to additional replicas. A v1 engine will not mark the final replica for a
-running volume as failed until twice the configured timeout. This behavior is intended to balance volume responsiveness
-with volume availability:
+This timeout only applies to additional replicas. A v1 engine will not mark the final replica for a running volume as
+failed until twice the configured timeout. This behavior is intended to balance volume responsiveness with volume
+availability:
```
Comment (jillian-maroket) [inline content/docs/1.7.1/references/settings.md:429]:
```suggestion
This setting only applies to additional replicas. A V1 engine marks the last active replica as failed only after twice the configured number of seconds (timeout value x 2) have passed. This behavior is intended to balance volume responsiveness with volume availability.
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

**Review by derekbit — APPROVED** [review]

**Pull request merged**